docs/order-validation.md

```diff
@@ -42,11 +43,34 @@ Your endpoint will receive a POST request with a JSON body content like this:
   {
     "CUSTOM_DOMAIN": "http://custom.doma.in"
   },
+  "buyerId": 111,
   "orderType": "NORMAL",
   "language": "en"
 }
 ```
 
+### Change plan
+
+```json
+{
+  "productId": 123,
+  "productIdentifier": "your-product",
+  "productVersionId": 789,
+  "productVersionIdentifier": "your-new-product-plan",
+  "billingItems":
+  {
+    "USERS": 10
+  },
+  "configurationParameters":
+  {
+    "CUSTOM_DOMAIN": "http://custom.doma.in"
+  },
+  "subscriptionId": 999,
+  "orderType": "SYNDICATED_UPGRADE",
```

Comment:
qui buyerId non c'è?

Reply:
in effetti sì, è un po' ridondante in questo caso vista la presenza di `subscriptionId` quindi non l'ho riportato